<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>655</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PDF TOOLS</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="PushButtonRiduci">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>230</y>
      <width>84</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Comprimi</string>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>71</y>
      <width>471</width>
      <height>64</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Vertical</enum>
    </property>
    <widget class="QLineEdit" name="Pdf1"/>
    <widget class="QLineEdit" name="Pdf2"/>
   </widget>
   <widget class="QPushButton" name="PushButtonSfoglia">
    <property name="geometry">
     <rect>
      <x>52</x>
      <y>230</y>
      <width>84</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Sfoglia</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButtonUnisci">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>230</y>
      <width>84</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Unisci</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButtonProteggi">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>230</y>
      <width>84</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Proteggi</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButtonRimuoviProtezione">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>230</y>
      <width>121</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Rimuovi Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="PasswordField">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>140</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_password">
    <property name="geometry">
     <rect>
      <x>92</x>
      <y>140</y>
      <width>101</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="FiligranaField">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>180</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_Filigrana">
    <property name="geometry">
     <rect>
      <x>92</x>
      <y>180</y>
      <width>101</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Filigrana</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButtonFiligrana">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>230</y>
      <width>84</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Filigrana</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_output_filename">
    <property name="geometry">
     <rect>
      <x>92</x>
      <y>110</y>
      <width>101</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Output filename</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_input_filename">
    <property name="geometry">
     <rect>
      <x>92</x>
      <y>80</y>
      <width>90</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Input filename</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButtonCancellaCampi">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>230</y>
      <width>110</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancella Campi</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
